### PR TITLE
Game quits when you collect 4 orbs. (WIP)

### PR DESCRIPTION
**Pull request by DSF256**
The game should quit whenever you collect 4 orbs (i.e., finish the level).

This is a WIP, obviously.  The game should move onto level 2, but I haven't completed that requirement yet.  Funny how we're still doing something related to waterfall even though we're using agile. :/

Btw, this took WAY too long for me to figure out how to do without breaking the rest of the game. :/

### CODE REVIEW

File: 2d/platformer/src/Actors/Player.gd [hunk 1]
```diff
@@ -155,6 +155,8 @@ func _on_coinCollected():
 func _on_orbCollected():
 	self.orbs = self.orbs + 1
 	$UI/amtOrbsLabel.text = "Orbs: " + String(self.orbs)
+	if (self.orbs >= 4):
```
Comment (reid-wynkoop) [inline 2d/platformer/src/Actors/Player.gd]: Should have a constant global var for the amount of orbs needed.

File: 2d/platformer/src/Actors/Player.gd [hunk 2]
```diff
@@ -169,7 +171,12 @@ func _collideWithEnemyCheck():
 					$UI/hpCanvasLayer/hpContainer/hpBar.value = self.curLife
 					self.contactWithEnemytimer.start()
 					_diedCheck()
-					
+
+func _orbCheck():
+	# for now, the game will just quit whenever you collect 4 orbs
+	# obviously, that'll get changed later
+	get_tree().quit()
```
Comment (reid-wynkoop) [inline 2d/platformer/src/Actors/Player.gd:178]: Agree, for now this is fine.

**Review by reid-wynkoop — APPROVED** [review]

File: 2d/platformer/src/Actors/Player.gd (hunk 1 above)
Comment (mbagwell) [reply 2d/platformer/src/Actors/Player.gd]: It says the code is outdated for Player.gd, might want to resolve that first.

Comment (reid-wynkoop) [reply 2d/platformer/src/Actors/Player.gd]: @mbagwell that just means he made a commit to it after I made a comment. In his commit, he created a constant var like I suggested, so it is fixed.

**Review by mbagwell — APPROVED** [review]
The orb function makes sense, when the user gets x amount of orbs the  game quits.

**Pull request merged**